apis/appmesh/v1beta2/types.go

```diff
@@ -226,3 +83,11 @@ type MeshReference struct {
 	// UID is the UID of Mesh CR
 	UID types.UID `json:"uid"`
 }
+
+// VirtualGatewayReference holds a reference to VirtualGateway.appmesh.k8s.aws
+type VirtualGatewayReference struct {
+	// Name is the name of Mesh CR
```

Comment:
nit: VirtualGateway instead of Mesh here and below.

Reply:
Copy-paste error. :) Will fix it.